README.adoc

```diff
@@ -1808,6 +1808,36 @@ date.all_quarter
 date.all_year
 ----
 
+=== Comparison
+
+Avoid comparing time with `>` and `<` to avoid confusion.
+
+[source,ruby]
+----
+# bad
+created_at < 5.minutes.ago
```

Comment:
A concern I have is that people are used to `<` in a SQL query involving dates, and so it feels natural to use it in Rails too.

I agree that `before?/after?` is a better and more natural API but it might be a step too far to mandate this.